examples/requirements.txt

```diff
@@ -0,0 +1 @@
+snowplow-tracker==0.10.0
```

Comment:
I think this shouldn't be necessary – the example app should use the snowplow_tracker package from the parent folder, not the published one. I don't think this requirement is being used at the moment (at least not in CI) but it is a bit confusing that it is here.